lib/api/init.php

```diff
@@ -18,7 +18,7 @@
 
 // Mode.
 if ( ! defined( 'SCRIPT_DEBUG' ) ) {
-	define( 'SCRIPT_DEBUG', false ); // @codingStandardsIgnoreLine - WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedConstantFound.
+	define( 'SCRIPT_DEBUG', false ); // @phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedConstantFound -- Valid use case as we need it defined.
```

Comment:
I'm surprised WPCS doesn't already allow for this exclusions.

Reply:
Me too.

Reply:
I've opened https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/1311

Reply:
Perfect. Once that rule is fixed, then we can remove the ignore above. Thank you.